btk/metrics.py

```diff
@@ -98,12 +98,11 @@ def meas_ksb_ellipticity(image, additional_params):
     gal_image = galsim.Image(image[meas_band_num, :, :])
     gal_image.scale = pixel_scale
     shear_est = "KSB"
-    try:
-        res = galsim.hsm.EstimateShear(gal_image, psf_image, shear_est=shear_est, strict=True)
-        result = [res.corrected_g1, res.corrected_g2, res.observed_shape.e]
-    except RuntimeError as e:
-        print(e)
-        result = [-10.0, -10.0, -10.0]
+
+    res = galsim.hsm.EstimateShear(gal_image, psf_image, shear_est=shear_est, strict=False)
+    result = [res.corrected_g1, res.corrected_g2, res.observed_shape.e]
+    if res.error_message != "":
+        print(res.error_message)
```

Comment:
I found your comment: 

> It happens when SExtractor makes spurious detections/only detects a part of a galaxy which is barely above the noise.

more enlightening than the default error message. Perhaps it would be good to also add some more information along these lines to the "Unphysical situation: galaxy convolved with PSF is smaller than PSF!" message? which might be confusing to some users.

Reply:
Something like 

> `galsim.hsm.EstimateShear` returned the error "Unphysical situation: galaxy convolved with PSF is smaller than PSF!", this might be caused by your measurement algorithm making spurious detections which are barely above the noise.